.circleci/config.yml

```diff
@@ -152,17 +160,74 @@ jobs:
   test_installation_from_source_develop_mode:
     # Test development installation
     docker:
-      - image: condaforge/mambaforge
+      - image: condaforge/mambaforge:latest
     resource_class: large
     steps:
       - test_installation_from_source:
           extra: develop
           flags: "--editable"
 
+  test_upstream_development:
+    # Test running recipes with the development version of ESMValCore. The
+    # purpose of this test to signal backward-incompatible changes early on in
+    # the development cycle.
+    docker:
+      - image: condaforge/mambaforge:latest
+    resource_class: large
+    steps:
+      - run:
+          name: Install git+ssh
+          environment:
+            DEBIAN_FRONTEND: noninteractive  # needed to install tzdata
+          command: apt update && apt install -y git ssh
```

Comment:
btw why is ssh needed? Doesn't git come with its own ssh version?

Reply:
CircleCI has a very convenient button called 'Rerun job with SSH' that you can use to log into the machine that is running the tests and poke around to see why they're failing. It only works if ssh is installed on that machine. The checkout step also uses ssh, but I think there might be some fallback option if it's missing.

Reply:
gotcha! Didn't know about that - I always rerun peasantly with the button